packages/database/src/schema.ts

```diff
@@ -224,7 +224,7 @@ export default [
       ['hash', 'String'],
       ['fee', 'String'],
       ['proof', 'Object'],
-      ['memo', 'String', { optional: true }],
+      ['memo', 'Object', { optional: true }],
```

Comment:
What do you think of splitting this into `memoVersion: string` and `memoData: string`?

The `memoData` could be the fully formed data (e.g. including the version data). The `memoVersion` would just be to make sorting more simple.

Then we'd just have a simple method for parsing the memo data and returning the structure (like with blocks).

Reply:
Resolved by 1594651